financialaid/constants.py

```diff
@@ -64,9 +64,13 @@ class FinancialAidStatus:
 )
 
 FINANCIAL_AID_DOCUMENTS_RECEIVED_MESSAGE = (
-    "We have received your documents verifying your income. We will review them as soon as possible, "
-    "after which we will send an e-mail confirming your personalized course price. We encourage you "
-    "to enroll now and pay later, when a decision has been reached."
+    "We have received your documents verifying your income. We will review and process them within "
+    "5 working days. If you have not received a confirmation email within one week, please feel free "
+    "to reach out to us at micromasters-support@mit.edu. Otherwise you should receive a confirmation "
```

Comment:
can you use env var `EMAIL_SUPPORT` from settings here? https://github.com/mitodl/micromasters/blob/master/micromasters/settings.py#L292